packages/wp-now/src/execute-wp-cli.ts

```diff
@@ -1,43 +1,137 @@
-import startWPNow from './wp-now';
 import { downloadWPCLI } from './download';
-import { disableOutput } from './output';
+import { rootCertificates } from 'tls';
 import getWpCliPath from './get-wp-cli-path';
 import getWpNowConfig from './config';
 import { DEFAULT_PHP_VERSION, DEFAULT_WORDPRESS_VERSION } from './constants';
-import { dirname } from 'path';
+import { phpVar } from '@php-wasm/util';
+import { createNodeFsMountHandler, loadNodeRuntime } from '@php-wasm/node';
+// import { getSqliteCommandPath } from '../../../src/lib/sqlite-command-versions';
+import {
+	PHP,
+	MountHandler,
+	writeFiles,
+	setPhpIniEntries,
+	loadPHPRuntime,
+} from '@php-wasm/universal';
```

Comment:
There are some lint errors, let's remove commented code and unused variables.

```suggestion
import {
	PHP,
	MountHandler,
	writeFiles,
	setPhpIniEntries,
} from '@php-wasm/universal';
```

Reply:
Linted.